docs/manuals/core/config/data-node-config.md

```diff
@@ -346,3 +346,43 @@ In this example, we configure an in_memory data node with the id "date", the sco
     the method `Config.configure_data_node()^` with parameter `storage_type="in_memory"`.
 
 [:material-arrow-right: The next section introduces the task configuration](task-config.md).
+
+# JSON
+
+An `JSONDataNode^` is a specific data node used to model JSON
```

Comment:
"An `JSONDataNode^` is a specific data node used to model JSON"
->
"A `JSONDataNode^` is a type of data node used to model JSON"

If JSONDataNodes can ONLY deal with JSON files, and not JSON data, when we're good.
Otherwise "file data" -> "data".